blocks/medium-promo-block/features/medium-promo/default.jsx

```diff
@@ -121,32 +250,35 @@ MediumPromo.propTypes = {
 			group: "Show promo elements",
 		}),
 		showByline: PropTypes.bool.tag({
-			label: "Show byline",
 			defaultValue: true,
+			label: "Show byline",
 			group: "Show promo elements",
 		}),
 		showDate: PropTypes.bool.tag({
-			label: "Show date",
 			defaultValue: true,
+			label: "Show date",
 			group: "Show promo elements",
 		}),
 		imageOverrideURL: PropTypes.string.tag({
 			label: "Image URL",
 			group: "Image",
 			searchable: "image",
 		}),
-		...imageRatioCustomField("imageRatio", "Art", "16:9"),
+		imageRatio: PropTypes.oneOf(["16:9", "3:2", "4:3"]).tag({
+			defaultValue: "3:2",
```

Comment:
Was 3:2 the default aspect ratio before? in the mocks it look like the default is 4:3

<img width="845" alt="Screen Shot 2022-08-29 at 14 22 29" src="https://user-images.githubusercontent.com/5950956/187281556-3d14d8ac-6c1a-4e25-8171-a04c5a734b5d.png">

Reply:
Default was `16:9` the last argument of the function is the default value

Reply:
yep

Reply:
@matthewroach , @JackHowa : Thank you! I've changed the default to `16:9`.